docs/Tutorials/GettingStarted.md

```diff
@@ -0,0 +1,403 @@
+  `-D BUILD_SHARED_LIBS=ON`. Note that the python bindings don't yet fully work
+  with shared library builds. All of the above together reduce the built total
+  size to approximately 1.5GB with Clang-11.
+- You can use the Ninja build generator by passing `-GNinja` to CMake. Ninja is
+  an alternative to Make that gives up some functionality but the simplification
+  allows for much faster dependency analysis leading to extremely quick
+  rebuilds.
+- Using ccache (compiler cache) allows for quick rebuilds when switching
+  branches. ccache caches the object files during a build allowing the cached
+  object to be used if the target is later triggered for a rebuild but a valid
+  cache is stored. The default ccache size is around 5GB. Increasing this to
+  20GB will help reduce build times if you manage several branches
+  simultaneously. The ccache cache size is specified by adding `max_size = 20G`
+  in `~/.ccache/ccache.conf`. ccache is enabled by default and can be disabled
+  using `-D USE_CCACHE=OFF`. Note that if ccache is not installed it cannot be
+  used.
+
+With over ~250,000 lines of C++, SpECTRE is no longer a small code base.
+Different parts of SpECTRE can be built separately to reduce how long it
+takes to get the functionality you need. For example, if you want to run a 1d
+plane wave scalar wave solution, you can only build the target
+`EvolveScalarWavePlaneWave1D`. To build only the unit tests, you can build the
+target `unit-tests`. To get a list of all the available targets, build
+`list`. You will get an alphabetical list of all the targets. All evolution
+executables are prefixed with `Evolve`, while elliptic solvers are prefixed with
+`Solve`. When building targets you should build with almost all of your cores or
+hyperthreads. For example, if you have a hyperthreaded dual-core CPU, then you
+can reasonable build on 3 threads in parallel. This leaves one thread for you to
+do other things while the code is building. Specifically, to build the unit
+tests on a hyperthreaded dual-core CPU you would run
+```
+make -j3 unit-tests
+```
+or
+```
+ninja -j3 unit-tests
+```
+if you are using Ninja instead of Make.
+
+Once you have built the unit tests you can run them using CTest as follows
+```
+ctest -L unit -j3
+```
+Notice that you can run the tests in parallel as well to help reduce wait time.
+
+You can build all the executables used in the tests by building the target
+`test-excutables`. This will build several evolution and elliptic solver
+executables and test that the example input files work. Note that building the
+executables can take a while and a single executable build can use 8GB of memory
+if you haven't disabled debug symbols. You can run all the tests using
+```
+ctest -j3
+```
+While the input file tests use a lot of memory to build, they don't require much
+memory to run, so running them in parallel is fine.
+
+# Running your first 1d evolution {#spectre_running_first_1d_evolution}
```

Comment:
This is a very helpful user guide!

This section is sufficiently long that it might be a good idea to break it up into subsections, like 'composing input files' and 'running and visualizing'